### PR TITLE
Set up a "Hello world" project with two apps

### PR DESCRIPTION
**Pull request by HENRYKC24**
## In this pull request, I did the following:point_down:
- Created a new Rails API app called 'hello-rails-back-end'.
- Initialized my project with Git.
- Made sure that my project has a Postgres database set up.
- Created a table for storing my messages and create 5 different greetings.
- Created an API endpoint that selects a random greeting from my table (I used a controller with an action and rails route)

### CODE REVIEW

File: app/controllers/v1/greetings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class V1::GreetingsController < ApplicationController
+  def index
+    @greetings = Greeting.find(Greeting.pluck(:id).sample)
```
Comment (rloterh) [inline app/controllers/v1/greetings_controller.rb:3]: - [ ] Your app is rendering a static screen, and hence no random greeting is being rendered when the _greet_ button is clicked or the page is refreshed. Kindly ensure to fix this issue

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Henry, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦   🚦  

## Good Points 👍  
- Descriptive Pull Request
- Good ReadMe
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/v1/greetings_controller.rb (hunk 1 above)
Comment (HENRYKC24) [reply app/controllers/v1/greetings_controller.rb:3]: Thank you very much:+1:. I had a CORS issue but it is solved now.

**Review by dasileker — APPROVED** [review]
## **Hi @HENRYKC24,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HENRYKC24** [comment]
Thank you very much:pray: I will proceed to merge as you directed:running:

**Pull request merged**